.github/workflows/sync.yml

```
@@ -0,0 +1,47 @@
+name: Sync All Branches with Upstream
+  permissions:
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 2: mapping values are not allowed in this context

(syntax-check)